scripts/Scripts/TestData/NewCollectionEventData.json

```diff
@@ -1,230 +1 @@
98-84e3-cce269c57faf}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":1,"CATCH_ID":28,"SPECIES_CODE":"CB","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{846934aa-6619-4764-8509-396b5b71739a}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":2,"CATCH_ID":1,"SPECIES_CODE":"ANY","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{becbf3d0-9dba-4087-81aa-98131d5367ca}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":2,"CATCH_ID":2,"SPECIES_CODE":"ANY","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{5db9404b-628f-4da9-8465-cdf879149bb0}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":2,"CATCH_ID":3,"SPECIES_CODE":"ANY","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{2d880a6a-52c8-4936-84d1-34425fea9898}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":3,"CATCH_ID":1,"SPECIES_CODE":"BKOW","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{88fe0223-a5e2-44ff-80fe-d4fe6cb0e7a6}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":3,"CATCH_ID":2,"SPECIES_CODE":"BKOW","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{26d96ac5-b755-4a2a-8635-4afe18af0b80}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":3,"CATCH_ID":3,"SPECIES_CODE":"BKOW","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{4097c6af-9a6f-491f-8957-609c05bdaf18}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":3,"CATCH_ID":4,"SPECIES_CODE":"CB","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{d0af5a0e-b00d-42c3-81e4-8f24483a676f}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":3,"CATCH_ID":5,"SPECIES_CODE":"CB","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{d32ce364-2778-45b5-893e-12fef1665982}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":3,"CATCH_ID":6,"SPECIES_CODE":"CB","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{d8233a85-5b6f-44e5-89d9-95d9df19f5f0}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":3,"CATCH_ID":7,"SPECIES_CODE":"CB","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{b149f441-8e49-4d88-84ac-82131522f7e0}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":3,"CATCH_ID":8,"SPECIES_CODE":"CB","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null},{"FISH_ID":"{bf6030f9-db70-4e96-8b6c-c5f8c30804f5}","EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","PASS_NUM":3,"CATCH_ID":9,"SPECIES_CODE":"CB","LENGTH_TYPE":null,"LENGTH":null,"WEIGHT":null}],"Diet":[],"Tags":[],"Health":[],"Habitat":[{"EVENT_ID":"{55be9e01-cd0f-419a-83f7-4856ace994b0}","BANKVEG":null,"BWID":null,"DEPTR":null,"DEPTL":null,"DEPM":null,"DEPMAX":null,"DOVR":"","DUND":"","LGWD":null,"POOL":null,"SPNG":"","RIFF":null,"RUNA":null,"SUB_FINES":null,"SUB_SAND":null,"SUB_GRAV":null,"SUB_COBB":null,"SUB_RUBB":null,"SUB_BOUL":null,"VEGD":null,"WWID":null,"SIN":null,"VEL":null,"EROS":null,"TEMP_":null,"PH":null,"CON":null}]}
```

Comment:
I believe that this object is also used in unit tests. I think that we will be looking at it more often in that regard rather that using it to publish the gp task. I would vote to keep this as prettified for that purpose. It's much easier to update with schema changes in a prettified state as well.

Reply:
reverted